docker/k8s/scripts/tcs_listener.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Any specific reason to duplicate this file for kubernetes ?

Reply:
Since `scripts/tcs_startup.sh` have managed the startup of both `tcs_listener` and `enclave_manager`, it's unfriendly for the k8s style of managing each component in different pods. Therefore, I have separate `tcs_listener` and `enclave_manager` into two scripts to make it possible to manage in k8s pod for scaling later

Reply:
Actually even for docker build, it make sense to separate tcs_listener and enclave_manager in different containers. Could you place these script at top level scripts folder. (enclave_manager.sh, tcs_listener.sh, lmdb.sh, propose_requests.sh). These scripts could be reused even for docker based builds

Reply:
That's great! Done as 016873573647a4b016bd21440390899263a8d171